src/server/controllers/cookieDetails.controller.js

```diff
@@ -0,0 +1,13 @@
+import config from '../config';
+
+// Index Route
+export const index = (req, res) => {
+  const language = req.i18n_lang;
+  if (config.isDevelopment()) {
```

Comment:
Seems like the end result of whether these conditions are true or false is the same, could we simplify to just call the render in any case?